README.md

```diff
@@ -46,12 +46,24 @@ Project documentation is available on the Jetty Eclipse website.
 Building
 ========
 
-To build, use:
+To build, use [Apache Maven 3.8.0](https://maven.apache.org/) (or better):
+
+Fast Build:
 
 ``` shell
-  mvn clean install
+mvn -Pci clean install -DskipTests
 ```
 
+Full Build with All Tests:
+
+``` shell
+mvn clean install
+```
+
+Optional build tools: 
+
+* [`graphviz`](https://graphviz.org/) - used by asciidoctor in the jetty-documentation build to produce various graphs
+
```

Comment:
maybe Docker can be added here
```suggestion
* [`Docker`](https://www.docker.com/) - used to run some integration tests for testing third party integrations
```

Reply:
@olamy go ahead and make the edits on this branch.

Might want to point out the version of JDK too?